frontend/src/v5/ui/v4Adapter/overrides/preview/previewComments.overrides.ts

```diff
@@ -20,7 +20,7 @@ import { Select, FilterWrapper, Container as CommentListContainer, Label as Show
 import { CommentContainer, Comment, Container as CommentAndDeleteButtonContainer } from '@/v4/routes/components/messagesList/components/message/components/userMessage/userMessage.styles';
 import { Container as CommentPadding } from '@/v4/routes/components/messagesList/components/message/message.styles';
 import { Counter, Actions, StyledTextField, ActionsGroup, Container as AddNewCommentContainer } from '@/v4/routes/viewerGui/components/commentForm/commentForm.styles';
-import { Collapsable, NotCollapsableContent, ToggleButtonContainer as CollaspableButton } from '@/v4/routes/viewerGui/components/previewDetails/previewDetails.styles';
+import { Collapsable, NotCollapsableContent, ToggleButtonContainer as CollaspableContainerButton } from '@/v4/routes/viewerGui/components/previewDetails/previewDetails.styles';
```

Comment:
I know this isn't exactly your fault as the component names are preexisting. But I find it funny that someone has been able to make two typos in the word 'Collapsible'